dags/error_aggregates.py

```diff
@@ -1,29 +1,39 @@
-import datetime
+# Generated via query_scheduling/generate_airflow_dags
 
-from airflow import models
+from airflow import DAG
+from airflow.operators.sensors import ExternalTaskSensor
+import datetime
 from utils.gcp import bigquery_etl_query
 
 default_args = {
-    'owner': 'bewu@mozilla.com',
-    'start_date': datetime.datetime(2019, 11, 1),
-    'email': ['telemetry-alerts@mozilla.com', 'bewu@mozilla.com', 'wlachance@mozilla.com'],
-    'email_on_failure': True,
-    'email_on_retry': True,
-    'depends_on_past': False,
-    'retries': 1,
-    'retry_delay': datetime.timedelta(minutes=20),
+    "owner": "bewu@mozilla.com",
+    "start_date": datetime.datetime(2019, 11, 1, 0, 0),
+    "email": [
+        "telemetry-alerts@mozilla.com",
+        "bewu@mozilla.com",
+        "wlachance@mozilla.com",
+    ],
+    "depends_on_past": False,
+    "retry_delay": datetime.timedelta(seconds=1200),
+    "email_on_failure": True,
+    "email_on_retry": True,
+    "retries": 1,
 }
 
-dag_name = 'error_aggregates'
-
-with models.DAG(
-        dag_name,
-        schedule_interval=datetime.timedelta(hours=3),
-        default_args=default_args) as dag:
+with DAG(
+    "bqetl_error_aggregates",
```

Comment:
Does changing the dag name remove the historical runs in the ui?  Doesn't matter in this case but worth noting.

Reply:
I would expect it does, yes. That's a good point to keep in mind as we transition, but in most cases I don't see any significant benefit to keeping it as long as recent runs have been successful.